packages/@pollyjs/persister/src/index.js

```diff
@@ -135,6 +136,10 @@ export default class Persister {
     );
   }
 
+  stringify() {
```

Comment:
Added a utility method here to packages extending this base class wont have to have `fast-json-stable-stringify` as a dependency. 

Not the biggest fan of this but its needed. @jasonmit lemme know if you have a better suggestion.

Reply:
Another option is @pollyjs/utils